substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/monitor/MultiThreadedMonitorSupport.java

```diff
@@ -474,7 +528,7 @@ protected JavaMonitor getOrCreateMonitorFromMap(Object obj, boolean createIfNotE
         }
     }
 
-    protected JavaMonitor newMonitorLock() {
+    protected static JavaMonitor newMonitorLock() {
```

Comment:
This method was made static not for the sake of inlining, but because it's now called in static context, from `private static void slowPathMonitorEnter()`